### ISSUE TITLE
CI failure: compilation error

### ISSUE THREAD

**Issue by ralf0131** [apache/dubbo#4164]
```
[INFO] -------------------------------------------------------------
[ERROR] COMPILATION ERROR : 
[INFO] -------------------------------------------------------------
[ERROR] /home/travis/build/apache/incubator-dubbo/dubbo-rpc/dubbo-rpc-thrift/src/main/java/org/apache/dubbo/rpc/protocol/thrift/ThriftInvoker.java:[99,26] error: cannot find symbol
  symbol:   method subscribeTo(CompletableFuture<Object>)
  location: variable asyncRpcResult of type AsyncRpcResult
```

**Comment by ralf0131** [comment]
I could not reproduce this error on my local environment.  However, I see twice in travis failure log... 
It looks weird.

**Comment by htynkn** [comment]
Iguess it fixed via #4158

**Comment by htynkn** [comment]
CI for master branch all green now. Close this issue
